src/azul/openapi/validation.py

```diff
@@ -113,15 +115,16 @@ def validate(self, request_parameter: Mapping[str, str]):
             schema_type = self.schema['type']
             if schema_type == 'string':
                 if self.schema.get('format') == 'uuid':
+                    # noinspection PyUnusedLocal
```

Comment:
I didn't intend to instruct you to add this `noinspection`. I don't think it's necessary since we don't ignore this warning in other places, but if you want to leave it in, go ahead.

Reply:
>Please address the first two warnings and the misspelling of parameters. The "shadowed name" warnings are fine to ignore I think.

I thought the `not used parameters` fell into the case of warnings to address.  It looks like there is other usage of this in the codebase, ill be keeping it in.